README.md

```diff
@@ -124,21 +124,12 @@ Option 1 Author your own test plan: Modify the `testPlan.fx.yaml` of a provided
 
 Option 2 Convert recorded tests from Test Studio: If you have tests that you have recorded in [Test Studio](https://docs.microsoft.com/en-us/power-apps/maker/canvas-apps/test-studio), you can convert your Test Studio tests to Test Engine.
 
-Note: Currently this is a slightly complicated process. In the near future we'll have the download button available in Test Studio to download the converted test plan.
-
-1. Download your Power Apps app.
-2. Rename your `.msapp` file by adding `.zip` at the end
-3. Unzip your zipped `.msapp` file and you will see a `AppTests` folder.
-4. In the folder there is a `2.json` file. Run the following commands in command line to convert an Test Studio json to a Test Engine yaml test plan.
-
-```bash
-# Convert the extracted Test Studio test to a Test Engine test plan
-dotnet run convert "path\to\yourApp.msapp.zip\AppTests\2.json"
-```
-
-5. Open the yaml file generated and add the logical name or app ID of your app. The steps to get them are [here](https://github.com/microsoft/PowerApps-TestEngine#remarks).
-6. Make sure you update the config file and user configurations if you are using a different tenant or environment for this app. You will need to modify `testPlanFile` with the path to the `2.fx.yaml` file for the sample that you wish to run.
-7. Now you should be ready to run the test with `dotnet run`
+1. Open the .msapp in Maker Portal.
+2. Navigate to the Test Studio.
+2. Click the download button available in Test Studio to download the converted test plan.
```

Comment:
nit: Duplicate second step

Reply:
Also, there are two ways to download the suite. One is to click download button in the command bar as mentioned here and another is to click elipisis sitting right next to each suite and then clicking download from the options menu that appear when elipisis are clicked